### ISSUE TITLE
Consider switching from XML to HTML

### ISSUE THREAD

**Issue by GoogleCodeExporter** [SeasideSt/Seaside#823]
```
This is here for tracking and discussion.

Currently we generate HTML in an XML compatible way. So we generate

<input checked="checked">

instead of

<input checked>
```

Original issue reported on code.google.com by `philippe...@gmail.com` on 23 Aug 2014 at 9:31

**Comment by marschall** [comment]
I think we should.

**Comment by marschall** [comment]
Fixed by #977
